driver-core/src/main/java/com/datastax/driver/core/AggregateMetadata.java

```diff
@@ -101,11 +106,29 @@ static AggregateMetadata build(KeyspaceMetadata ksm, Row row, VersionNumber vers
         if (version.getMajor() >= 3) {
             stateType = DataTypeCqlNameParser.parse(stateTypeName, cluster, ksm.getName(), ksm.userTypes, null, false, false);
             String rawInitCond = row.getString("initcond");
-            initCond = rawInitCond == null ? null : codecRegistry.codecFor(stateType).parse(rawInitCond);
+            if (rawInitCond == null) {
+                initCond = null;
+            } else {
+                try {
+                    initCond = codecRegistry.codecFor(stateType).parse(rawInitCond);
```

Comment:
Which Cassandra version did you test against? IIRC, some versions between 3.0 and 3.4 use string literals, then at some point it was changed to a blob.